src/observability/publisher.go

```diff
@@ -41,7 +50,51 @@ func ProcessSystemSummary() {
 			// clear each published entry from data map
 			delete(tempSummarizerMap, ss)
 		}
+		log.Info().Msgf("Published %v events in %v", count, time.Since(timeVar))
 		PublisherMutex.Unlock()
 	}
 	ProcessSystemSummaryWg.Wait()
 }
+
+func getAggregatedSummaryMap(tempSummarizerMap map[types.SystemSummary]types.SysSummaryTimeCount) map[types.SystemSummary]types.SysSummaryTimeCount {
+
+	var fileArr []string
+	log.Info().Msgf("Events before aggregation: [%v]", len(tempSummarizerMap))
+
+	fileSumMap := make(map[types.SystemSummary]types.SysSummaryTimeCount)
+	updatedSummarizerMap := make(map[types.SystemSummary]types.SysSummaryTimeCount)
+	for tss, tsstc := range tempSummarizerMap {
+		if tss.Operation == types.FileOperation {
+			fileArr = append(fileArr, tss.Destination)
```

Comment:
We're using a common file array here and appending file paths from different pods and aggregating paths across all the pods. This is not correct. 

We should only run the aggregate path logic on events from a single path.